src/index.jsx

```diff
@@ -176,6 +187,7 @@ initialize({
         PRIVACY_POLICY_URL: process.env.PRIVACY_POLICY_URL || null,
         SHOW_UNGRADED_ASSIGNMENT_PROGRESS: process.env.SHOW_UNGRADED_ASSIGNMENT_PROGRESS || false,
         ENABLE_XPERT_AUDIT: process.env.ENABLE_XPERT_AUDIT || false,
+        OPENEDX_AI_SOCKET_DOMAIN: process.env.OPENEDX_AI_SOCKET_DOMAIN || '',
```

Comment:
[question]: Should there be a null here, similar to other config variables?

```suggestion
        OPENEDX_AI_SOCKET_DOMAIN: process.env.OPENEDX_AI_SOCKET_DOMAIN || null,
```